tests/core/block/test_naive_block.py

```diff
@@ -100,3 +100,42 @@ def test_get_num_free_blocks(allocate_type: str, num_blocks: int,
         for i, block in enumerate(blocks):
             assert allocator.get_num_free_blocks() == i
             allocator.free(block)
+
+    @staticmethod
+    @pytest.mark.parametrize("num_blocks", [4])
+    @pytest.mark.parametrize("block_size", [8])
+    def test_get_num_blocks_touched(num_blocks, block_size):
```

Comment:
Can you add a docstring explaining test methodology (e.g. what the test tests and how it does it)

Reply:
done